app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError, VaccineError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    not_wearing_mask = []
+    counter = 0
+    for friend in friends:
+        counter += 1
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError as e:
+            not_wearing_mask.append(e)
```

Comment:
List of exceptions..
Looks a bit weird. You can just increment the masks counter